src/ReplicatedStorage/Client/InstanceComponents/Misc/FallDamage.lua

```diff
@@ -6,19 +6,12 @@ local FallDamage = {}
 FallDamage.__index = FallDamage
 
 function FallDamage:_trackCharacterFall(character: Model, humanoid: Humanoid)
-	local initialHeight = nil
-	local fallHeight = nil
-
-	self._trove:Connect(humanoid.FreeFalling, function(active)
-		if active then
-			initialHeight = character.HumanoidRootPart.Position.Y
-		elseif initialHeight then
-			fallHeight = initialHeight - character.HumanoidRootPart.Position.Y
-			if fallHeight < FallDamageConfig.Threshold then return end
-
-			local damage = (fallHeight - FallDamageConfig.Threshold) * FallDamageConfig.Scale
-			humanoid:TakeDamage(damage)
-		end
+	local endVelocity = nil
```

Comment:
What is the purpose of this variable being local to the outside scope? Isn't it only used within the function?